.github/workflows/actions.yml

```diff
@@ -0,0 +1,64 @@
+name: Publish Python 🐍 distributions 📦 to PyPI
+on:
+  push:
+    tags:
+      - '*'
+env:
+  PYPI_USER: __token__
+  PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+  DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+jobs:
+  test-multiple-python-versions-and-extras:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # 4 jobs total with all permutations
+        python: [2.7, 3.5]
+        extras: ["test", "test,queable,sentry"]
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Test
+        working-directory: ./client
+        run: |
+          pip install -e .[${{ matrix.extras }}]
+          py.test
+  test-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Test
+        run: |
+          echo ${{ env.DOCKER_PASSWORD }} | docker login --username $DOCKER_USERNAME --password-stdin
+          make test
+  publish-to-pypi:
+    # only publish if all tests pass
+    needs: [test-multiple-python-versions-and-extras, test-docker]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: 3.5
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Create distribution
+        working-directory: ./client
+        run: |
+          pip install twine
+          pip install build
+          python -m build --sdist
+          python -m build --wheel
+      - name: Publish to PyPI
+        working-directory: ./client
+        run: |
+          twine upload -r pypi -u ${{ env.PYPI_USER }} \
+          -p ${{ env.PYPI_TOKEN }} dist/*
```

Comment:
Will the envvars be echoed in logs? I'm guessing that github actions are smart enough to prevent this, but I've never used them before.

Reply:
Correct, they are masked as "***" anytime they are in the logs. More info can be found at https://docs.github.com/en/actions/security-guides/encrypted-secrets